src/App.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import './App.css';
+
+import users from './api/users';
+import todos from './api/todos';
+
+import { TodosList } from './components/todosList';
+
+const TodosWithUsers = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId).name,
+  isVisible: true,
+}));
+
+class App extends React.Component {
+  state = {
+    renderList: [...TodosWithUsers],
+    title: '',
+    user: '',
+    completed: false,
+    taskError: true,
+    userError: true,
```

Comment:
I would change logic
`error === true` should mean that there is indeed an error present.
```suggestion
    taskError: false,
    userError: false,
```